synapse/config/cas.py

```diff
@@ -57,6 +57,8 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
                 required_attributes
             )
 
+            self.cas_enable_registration = cas_config.get("enable_registration")
```

Comment:
For backwards compatibility we need to set this to `true`:

```suggestion
            self.cas_enable_registration = cas_config.get("enable_registration", True)
```